builtin/providers/aws/resource_aws_elastic_transcoder_pipeline.go

```diff
@@ -0,0 +1,393 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"regexp"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/elastictranscoder"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsElasticTranscoderPipeline() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsElasticTranscoderPipelineCreate,
+		Read:   resourceAwsElasticTranscoderPipelineRead,
+		Update: resourceAwsElasticTranscoderPipelineUpdate,
+		Delete: resourceAwsElasticTranscoderPipelineDelete,
+
+		Schema: map[string]*schema.Schema{
+			"arn": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
```

Comment:
Looks like this is an output only field? Nix `Optional: true` to make it read-only.